test/unit/test_events.py

```diff
@@ -45,3 +46,32 @@ def test_formatting(self):
         # were no args passed after the initial message
         event = AdapterEventDebug(name="dbt_tests", base_msg="boop{x}boop", args=())
         self.assertTrue("boop{x}boop" in event.message())
+
+class TestEventCodes(TestCase):
+
+    def get_all_subclasses(self,cls):
+        all_subclasses = []
+        for subclass in cls.__subclasses__():
+            all_subclasses.append(subclass)
+            all_subclasses.extend(self.get_all_subclasses(subclass))
+        return set(all_subclasses)
+
+
+    def setUp(self):
+        pass
+
+    # this interface is documented for adapter maintainers to plug into
+    # so we should test that it at the very least doesn't explode.
```

Comment:
This comment is probably a copy paste error. Let's write a new one that describes what this tests for and why it's important.